### DISCUSSION TITLE
Can I simulate a booking system?

### DISCUSSION THREAD

**Discussion by alevangel** [synthetichealth/synthea#1270] [Q&A] [answered]
Can I simulate a visit booking system? How to do that?
In particular, I'd be interested in simulating that patients can book a visit/consultation/therapy, so I'd like to be able to simulate that they: cancel/show up/don't show up.

The aim is to have a dataset more oriented towards governance than towards patient health itself. (something like SimPy style)

**Comment by jawalonoski** [comment, marked as answer]
No, Synthea does not include a scheduling system.

My recommendation would be for you to post-process the Encounter data, to add Appointments corresponding to the Encounters that took place, and insert random Encounters that were cancelled or rescheduled, etc.
